src/main/kotlin/blackjack/domain/Player.kt

```diff
@@ -0,0 +1,41 @@
+package blackjack.domain
+
+import blackjack.domain.card.Card
+
+class Player(val name: String) {
+    private val cards = mutableListOf<Card>()
+    private val cardNumberCalculator = CardNumberCalculator()
+
+    fun addCard(card: Card) {
+        cards.add(card)
+    }
+
+    fun addCards(drawCards: List<Card>) {
+        drawCards.forEach {
+            cards.add(it)
+        }
```

Comment:
```suggestion
        cards.addAll(drawCards)
```
addAll을 사용하시면 간단하게 전체 리스트를 추가할 수 있습니다.

Reply:
addAll을 나중에 추가해서 이런 코드가 발생한것 같네요 🥲 
꼼꼼하게 코드를 점검해보는 습관을 만들어야겠습니다 ✔️